part-spec/pinPaths.json

```diff
@@ -0,0 +1,42 @@
+{
+  "$id": "https://github.com/edatasheets/edatasheets.github.io/blob/main/part-spec/pinPaths.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "pinPaths": {
+    "type": "object",
+    "required": [
+      "partPinPaths"
+    ],
+    "properties": {
+      "numberofpinPaths": {
```

Comment:
I'm not sure if this is necessary since you can just count the number of parts in the array, but we can leave it as an optional field if you think it's helpful

Reply:
Good point. Yes this is optional and can be used to check against what we have in the arrays.